cvat/apps/iam/tests/adapters.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2022 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
```

Comment:
I will recommend to create a directory under /tests/python/social_auth and put all auxiliary code into it.